toolbackup/src/main/resources/awsConfig.conf

```diff
@@ -0,0 +1,2 @@
+AWS_ACCESS_KEY: AKIAIOSFODNN7EXAMPLE
```

Comment:
Could you reverse the order here and put `EXAMPLEKEY` out in the front and put maybe a underscore in to make this really obvious?  e.g. "EXAMPLE_KEY_AKIAIOS..."